src/models/spell/index.js

```diff
@@ -3,13 +3,13 @@ const Schema = mongoose.Schema;
 const { APIReference } = require('../common');
 
 const AreaOfEffect = new Schema({
-  _id: false,
-  size: Number,
-  type: String,
+  _id: { type: String, select: false },
```

Comment:
I think something is up with the typing of these `_id` fields on the subdocuments. When run this branch locally I'm seeing some error messages like ` Cannot do exclusion on field area_of_effect._id in inclusion projection` 🤔

Reply:
I think the proper change is to drop `_id` from subdocuments

Reply:
Yup! That fixed it!